src/dronevis/drone_connect/command.py

```diff
@@ -69,85 +91,81 @@ def configure(self, argument: str, value: str) -> bool:
             bool: flag for valid config operations
         """
         # Check if it's the first time we send a config
-        DEBUG = True
-        if not self.is_configurated:
+        if not self.is_configured:

```

Comment:
I think is_configured should be referenced using the class name instead of self as it is class variable